From 4d546d65b6ac4dc759aa688750718dab4f53f113 Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Sun, 27 Dec 2020 15:27:44 +0000
Subject: [PATCH 114/118] dt-bindings: arm: amlogic: add Bananapi BPI-M5
 bindings

Add the board bindings for the Sinovoip (Bananapi) BPI-M5 device.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 Documentation/devicetree/bindings/arm/amlogic.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/amlogic.yaml b/Documentation/devicetree/bindings/arm/amlogic.yaml
index 415fc1d351e1..ec93a9d664bc 100644
--- a/Documentation/devicetree/bindings/arm/amlogic.yaml
+++ b/Documentation/devicetree/bindings/arm/amlogic.yaml
@@ -173,6 +173,7 @@ properties:
               - hardkernel,odroid-hc4
               - khadas,vim3l
               - seirobotics,sei610
+              - sinovoip,bananapi-m5
           - const: amlogic,sm1
 
       - description: Boards with the Amlogic Meson A1 A113L SoC
-- 
2.31.1

